framework/src/controller/bus.ts

```diff
@@ -107,39 +108,17 @@ export class Bus {
 				port: config.rpc.port,
 				logger: this.logger,
 			});
-			this._wsServer.start();
 		}
 	}
 
 	public async setup(): Promise<boolean> {
-		if (!this.config.ipc.enabled) {
-			return true;
+		if (this.config.ipc.enabled) {
```

Comment:
It could be in this PR or not, but i think we should deprecate `ipc.enabled`.
the `rpc` config has `enable: true | false` and also `mode: 'ws' | 'ipc'` and we should use this so that we can increase the protocol if we need to

Reply:
Related to that we discussed earlier, if I am not missing it. The `IPC` will remain as internal communication structure regardless if WS is enabled or not. Let's discuss this further and then decide upon this change.